plugins/scaffolder-react/src/components/types.ts

```diff
@@ -56,5 +56,6 @@ export type ReviewStepProps = {
     uiSchema: UiSchema;
     mergedSchema: JsonObject;
     schema: JsonObject;
+    title: string;
```

Comment:
Unfortunately, this has to be added, or the title in `ParsedTemplateSchema` must be changed to allow undefined. Otherwise, the separation of the default review step is not possible because of TypeScript errors.